tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -39,14 +39,26 @@ function get_container_runtime() {
                 die "invalid node name"
 	fi
 	if echo "$runtime" | grep -qE 'containerd.*-k3s'; then
+		echo $(k3s_service_name)
+	else
+		echo "$runtime" | awk -F '[:]' '{print $1}'
+	fi
+}
+
+function k3s_service_name() {
+	local k3s_service=""
+	if [ -x /sbin/openrc-run ]; then
+		k3s_service=$(ls /etc/init.d/ | grep k3s)
```

Comment:
This feel error prone, in case something else k3s gets added there.
Do the services have a specific name that happens to be distro agnostic with OpenRC?

Reply:
agree, I wasn't sure here as well. In my setup, it is named as k3s-service. If checking with k3s official site, it dose name like this everywhere. But you never know if someone re-pack and re-distribute k3s and pick up another name.

Reply:
@jodh-intel, @devimc, may I borrow your brain here a little bit?  Do you have any suggestion on this?